.gitmodules

```diff
@@ -180,3 +180,6 @@
 [submodule "contrib/stats"]
 	path = contrib/stats
 	url = https://github.com/kthohr/stats.git
+[submodule "contrib/eigen"]
+	path = contrib/eigen
+	url = https://gitlab.com/libeigen/eigen.git
```

Comment:
eigen has some GPL parts as well, we have to define EIGEN_MPL2_ONLY

https://gitlab.com/libeigen/eigen/-/blob/master/COPYING.README